front/src/modules/ui/tables/hooks/useSetTableSoftFocusActiveStatus.ts

```diff
@@ -0,0 +1,18 @@
+import { useRecoilCallback } from 'recoil';
+
+import { isSoftFocusOnCellFamilyState } from '../states/isSoftFocusOnCellFamilyState';
+import { softFocusPositionState } from '../states/softFocusPositionState';
+
+export function useSetTableSoftFocusActiveStatus() {
+  return useRecoilCallback(({ set, snapshot }) => {
+    return (enabled: boolean) => {
+      const currentPosition = snapshot
+        .getLoadable(softFocusPositionState)
+        .valueOrThrow();
+
+      set(isSoftFocusOnCellFamilyState(currentPosition), false);
+      set(softFocusPositionState, currentPosition);
```

Comment:
Why setting the position to its current value ? 

Also why is it needed to use a recoilCallback ?